app/chrome/helpers/format.es6

```diff
@@ -8,9 +8,9 @@ let thresholds = [
 ];
 
 function format_bytes (bytes) {
-    for (var [label, min] in thresholds) {
+    for (let [label, min] of thresholds) {
       if (bytes >= min) {
-        return `${_str.numberFormat(bytes / min)}${label}`;
+        return `${_str.numberFormat(bytes / min)} ${label}`;
```

Comment:
I added a space here to match with https://github.com/elisee/itchio-app/blob/fix-coffee-and-nits/app/chrome/helpers/format.es6#L17 below but maybe this was on purpose. Let me know if you don't like the change.